### PR TITLE
Add technical and business owners to Service export

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/export_to_ng/resources.py [hunk 1]
```diff
@@ -579,6 +579,20 @@ class ServiceResource(resources.ModelResource):
     name = fields.Field(column_name='name')
     profit_center = fields.Field(column_name='profit_center')
     uid = fields.Field(column_name='uid')
+    technical_owners = fields.Field(column_name='technical_owners')
+    business_owners = fields.Field(column_name='business_owners')
+
+    def dehydrate_technical_owners(self, service):
+        technical_owners = service.owners.filter(
+            ciownership__type=models_ci.CIOwnershipType.technical
+        ).values_list('profile__user__username')
```
Comment (mkurek) [inline src/ralph/export_to_ng/resources.py]:
```
.values_list('profile__user__username', flat=True)
return ','.join(technical_owners)
```

File: src/ralph/export_to_ng/resources.py [hunk 2]
```diff
@@ -579,6 +579,20 @@ class ServiceResource(resources.ModelResource):
...
+        return ','.join([i[0] for i in technical_owners])
+
+    def dehydrate_business_owners(self, service):
+        technical_owners = service.owners.filter(
+            ciownership__type=models_ci.CIOwnershipType.business
+        ).values_list('profile__user__username')
+        return ','.join([i[0] for i in technical_owners])
```
Comment (mkurek) [inline src/ralph/export_to_ng/resources.py]: ditto

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**